Modules/Devices/src/Devices.Application/IntegrationEvents/Incoming/IdentityDeletionProcessStarted/IdentityDeletionProcessStartedIntegrationEventHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
+using Backbone.BuildingBlocks.Application.PushNotifications;
+using Backbone.Modules.Devices.Application.IntegrationEvents.Outgoing;
+
+namespace Backbone.Modules.Devices.Application.IntegrationEvents.Incoming.IdentityDeletionProcessStarted;
+
+public class IdentityDeletionProcessStartedIntegrationEventHandler : IIntegrationEventHandler<IdentityDeletionProcessStartedIntegrationEvent>
+{
+    private readonly IPushNotificationSender _pushNotificationSender;
+
+    public IdentityDeletionProcessStartedIntegrationEventHandler(IPushNotificationSender pushNotificationSender)
+    {
+        _pushNotificationSender = pushNotificationSender;
+    }
+
+    public async Task Handle(IdentityDeletionProcessStartedIntegrationEvent @event)
+    {
+        var payload = new
+        {
+            Type = "IdentityDeletionProcessStarted",
+            Message = "Deletion process started"
+        };
```

Comment:
The sequence diagram was a bit simplified in this place. I hoped you would look at similar cases (e.g. ExternalEventCreatedIntegrationEventHandler) to see how to send a push notification.